### PR TITLE
update(registry): add `k8smeta` plugin to the registry

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

/area registry

**What this PR does / why we need it**:

This PR adds `k8smeta` plugin to the registry

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: registry.yaml [hunk 1]
```diff
@@ -341,3 +341,27 @@ plugins:
...
+      cosign:
+        certificate-oidc-issuer: https://token.actions.githubusercontent.com
+        certificate-identity-regexp: https://github.com/falcosecurity/plugins/
+    capabilities:
+      extraction:
+        supported: true
+      parsing:
+        supported: true
+      async:
+        supported: true
```
Comment (Andreagit97) [inline registry.yaml:367]: not clear to me which are the supported values under `capabilities`, do we have a validation for this field?

Comment (leogr) [reply registry.yaml:367]: I guess we don't have it yet.

Comment (leogr) [reply registry.yaml:367]: We don't have it yet :point_down: 
https://github.com/falcosecurity/plugins/blob/master/build/registry/pkg/registry/types.go#L39

Could you remove `async` at the moment, please? We can still introduce it later.

Comment (Andreagit97) [reply registry.yaml]: done I used just `extraction` and added the `sources` field. Thank you for the pointer!

**Pull request merged**